common/config/rush/pnpm-config.json

```diff
@@ -29,7 +29,7 @@
    * The default value is false to avoid legacy compatibility issues.
    * It is strongly recommended to set `strictPeerDependencies=true`.
    */
-  "strictPeerDependencies": true,
+  "strictPeerDependencies": false,
```

Comment:
won't we face compatibility issues if we set this to false. Do we need to change the setting?

Reply:
Reverted